contracts/governor/GovernorBravoInterfaces.sol

```diff
@@ -0,0 +1,208 @@
+// SPDX-License-Identifier: Apache-2.0
+// Compound https://github.com/compound-finance/compound-protocol/blob/master/contracts/Governance/GovernorBravoInterfaces.sol
+pragma solidity ^0.8.0;
+
+contract GovernorBravoEvents {
+    /// @notice An event emitted when a new proposal is created
+    event ProposalCreated(
+        uint256 id,
+        address proposer,
+        address[] targets,
+        uint256[] values,
+        string[] signatures,
+        bytes[] calldatas,
+        uint256 startBlock,
+        uint256 endBlock,
+        string description
+    );
+
+    /// @notice An event emitted when a vote has been cast on a proposal
```

Comment:
suggetsion(blocking): be consistent with multi-line comments, either use `/** ... */` or use `///`

Reply:
I will replace it, but those comments are from Compound code.